mfthomps/CyberCIEGE-port-to-Unity3d
# Issue/172 attack log look and feel

Attack Log now flashes red when there is a new attack message.

To test:

- open the Introductory Scenario and complete Phase 1
- Press the play button and wait for an attack
- notice the red flashing attack log button

Closes #172
